BookPlayer/Library/FileManagement/ImportOperation.swift

```diff
@@ -21,15 +22,67 @@ class ImportOperation: Operation {
         self.files = files
     }
 
+    func handleZip(file: FileItem) {
+        // Unzip to a temp directory
+        if file.originalUrl.pathExtension == "zip" {
+            let tempURL = file.destinationFolder.appendingPathComponent("tmp")
+
+            do {
+                try FileManager.default.createDirectory(at: tempURL, withIntermediateDirectories: true, attributes: nil)
+                try FileManager.default.unzipItem(at: file.originalUrl, to: tempURL)
+
+            } catch {
+                print("Extraction of ZIP archive failed with error:\(error)")
+            }
+
+            do {
+                let fileManager = FileManager.default
+
+                do {
+                    let resourceKeys: [URLResourceKey] = [.creationDateKey, .isDirectoryKey]
+                    let enumerator = fileManager.enumerator(at: tempURL,
+                                                            includingPropertiesForKeys: resourceKeys,
+                                                            options: [.skipsHiddenFiles], errorHandler: { (url, error) -> Bool in
+                                                                print("directoryEnumerator error at \(url): ", error)
+                                                                return true
+                    })!
+
+                    for case let fileURL as URL in enumerator {
+                        if fileURL.pathExtension == "mp3" || fileURL.pathExtension == "m4a" || fileURL.pathExtension == "m4b" {
+                            let documentsURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!
+                            let destinationURL = documentsURL.appendingPathComponent(fileURL.lastPathComponent)
+                            try FileManager.default.moveItem(at: fileURL, to: destinationURL)
+                        }
+                    }
+                } catch {
```

Comment:
since we're not doing anything in this `catch` we can remove the `do` `catch` and let `FileManager.default.moveItem` fail silently using `try? FileManager.default.moveItem `